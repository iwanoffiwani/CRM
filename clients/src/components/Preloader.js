import React from 'react'
import LinearProgress from '@material-ui/core/LinearProgress';

const Preloader = props => (
  <div className='preloader'>
    <LinearProgress />
  </div>
)

export default Preloader