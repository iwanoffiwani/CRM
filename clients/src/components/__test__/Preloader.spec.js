import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Preloader from '../Preloader'

describe(`<Preloader />`, () => {
  it(`+++ renderer component`, () => {
    const wrapper = shallow(<Preloader />)
    return expect(toJson(wrapper)).toMatchSnapshot()
  })
})