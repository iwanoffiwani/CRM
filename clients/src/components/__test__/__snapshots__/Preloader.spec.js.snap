// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Preloader /> +++ renderer component 1`] = `
<div
  className="preloader"
>
  <WithStyles(ForwardRef(LinearProgress)) />
</div>
`;
